test/functional/terminal/job_spec.lua

```diff
@@ -31,9 +30,6 @@ describe('job handler', function()
         elseif 'stderr' ==# a:event
           put =join(a:data,"\n")
         elseif 'exit' ==# a:event
-          " Length of received data.
-          call setline('.', 'bytes: '.len(join(g:job_stdout[0:-2],"\n")))
```

Comment:
@oni-link This was unreliable, even after merging #4646. It varied by about 3 bytes. Is it fair to say this could be due to shell output inconsistencies?